www/js/lib/uiUtil.js

```diff
@@ -20,27 +20,65 @@
  * along with Kiwix (file LICENSE-GPLv3.txt).  If not, see <http://www.gnu.org/licenses/>
  */
 'use strict';
-define([], function() {
 
+// DEV: Put your RequireJS definition in the rqDef array below, and any function exports in the function parenthesis of the define statement
+// (This method is needed in order to load WebP polyfills conditionally)
+var rqDef = [];
+
+// Add WebP polyfill only if webpHero was loaded in init.js
+if (webpMachine) {
+    rqDef.push('webpHeroBundle');
+}
```

Comment:
Do we need to detect whether the app is in SW mode, and avoid adding the polyfills if that is the case? We certainly need to test whether loading polyfills via RequireJS conflicts with them being loaded in the iframe by the ZIM in SW mode. There *shouldn't* be a conflict because the iframe `contentWindow` is isolated from the `window` object. If we did disable polyfills in SW mode, it would imply that a user switching to jQuery mode might need to exit the app and reload it. (We can detect SW mode easily by scanning document.cookie.)

Reply:
OK, I agree we need to test in an old Firefox before merging.

Reply:
This has been tested, and an issue (possible race condition) unrelated to this PR has emerged, which will be dealt with in #673. I tested that the issue is unrelated to this PR by running Firefox 64 Potable against master (i.e. without this Polyfill) in SW mode, and randomly cycling though pages until the error occurs. It may take several page loads, but eventually images fail on a page.